doc_source/external-snat.md

```diff
@@ -5,15 +5,16 @@ If you deployed your cluster using the `IPv6` family, then the information in th
 By default, each pod in your cluster is assigned a [private](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-instance-addressing.html#concepts-private-addresses) `IPv4` address from a classless inter\-domain routing \(CIDR\) block that is associated with the VPC that the pod is deployed in\. Pods in the same VPC communicate with each other using these private IP addresses as end points\. When a pod communicates to any `IPv4` address that isn't within a CIDR block that's associated to your VPC, the [Amazon VPC CNI plugin for Kubernetes](https://github.com/aws/amazon-vpc-cni-k8s#amazon-vpc-cni-k8s) translates the pod's `IPv4` address to the primary private `IPv4` address of the primary [elastic network interface](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-eni.html#eni-basics) of the node that the pod is running on, by default [\*](#snat-exception)\.
 
  Due to this behavior:
-+ Resources that are in networks or VPCs that are connected to your cluster VPC using [VPC peering](https://docs.aws.amazon.com/vpc/latest/peering/what-is-vpc-peering.html), a [transit VPC](https://docs.aws.amazon.com/whitepapers/latest/aws-vpc-connectivity-options/transit-vpc-option.html), or [Direct Connect](https://docs.aws.amazon.com/directconnect/latest/UserGuide/Welcome.html) can't initiate communication to your pods\. Your pods can initiate communication to those resources and receive responses from them though\.
 + Your pods can communicate with internet resources only if the node that they're running on has a [public](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-instance-addressing.html#concepts-public-addresses) or [elastic](https://docs.aws.amazon.com/vpc/latest/userguide/vpc-eips.html) IP address assigned to it and is in a public subnet\. A public subnet's associated [route table](https://docs.aws.amazon.com/vpc/latest/userguide/VPC_Route_Tables.html) has a route to an internet gateway\. We recommend deploying nodes to private subnets, whenever possible\.
++ For VPC CNI versions earlier than 1.8.0, Resources that are in networks or VPCs that are connected to your cluster VPC using [VPC peering](https://docs.aws.amazon.com/vpc/latest/peering/what-is-vpc-peering.html), a [transit VPC](https://docs.aws.amazon.com/whitepapers/latest/aws-vpc-connectivity-options/transit-vpc-option.html), or [Direct Connect](https://docs.aws.amazon.com/directconnect/latest/UserGuide/Welcome.html) can't initiate communication to your pods\. Your pods can initiate communication to those resources and receive responses from them though\.
```

Comment:
nit: `Resources` -> `resources`

let's use bold and change to `For VPC CNI versions *less than v1.8.0*`

Let's also change "can't initiate communication to your pods" to "cannot initiate communication to your pods behind non-primary ENIs."

nit: "from them though" -> "from them, though"